src/main.ts

```diff
@@ -1,14 +1,28 @@
-// The Vue build version to load with the `import` command
-// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import 'babel-polyfill';
 import Vue from 'vue';
-import App from '@/App';
+import 'vue-material/dist/vue-material.min.css';
+import {
+  MdToolbar,
+  MdButton,
+  MdIcon,
+} from 'vue-material/dist/components';
+
+import App from '@/App.vue';
 import router from './router';
+import '@/views/components';
+import store from '@/vuex';
 
 Vue.config.productionTip = false;
 
+// Use material components
+Vue.use(MdToolbar);
+Vue.use(MdButton);
+Vue.use(MdIcon);
```

Comment:
Adding each material component individually to reduce bundle size. Should consider moving this into it's own file as it will get bloated